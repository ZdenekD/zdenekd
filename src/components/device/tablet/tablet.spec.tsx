import {shallow, ShallowWrapper} from 'enzyme';
import findComponent from '@/__test__/utils/helpers';
import Tablet from '.';

describe('Tablet device', () => {
    let wrapper: ShallowWrapper;

    beforeEach(() => {
        wrapper = shallow(<Tablet />);
    });

    it('match snapshot', () => {
        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders without error', () => {
        const component = findComponent(wrapper, 'component-device-tablet');

        expect(component.exists()).toBe(true);
    });
});
