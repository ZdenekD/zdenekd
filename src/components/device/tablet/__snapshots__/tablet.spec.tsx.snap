// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tablet device match snapshot 1`] = `
<div class="wrapper"
     data-test="component-device-tablet"
>
  <div class="tablet">
    <div class="content">
    </div>
    <span class="header">
      <i class="camera">
      </i>
    </span>
    <span class="footer">
      <i class="button">
      </i>
    </span>
  </div>
</div>
`;
